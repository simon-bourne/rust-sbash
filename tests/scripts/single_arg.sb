pub fn test-fn(arg1) {
    echo "$arg1"
}
