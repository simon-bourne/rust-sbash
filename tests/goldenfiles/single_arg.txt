test-fn () { ( arg1="$1"; shift; 
    echo "$arg1"
) };